pyproject.toml

```diff
@@ -1,17 +1,73 @@
 [tool.poetry]
-name = "pyap"
+name = "pyap-beauhurst"
 version = "0.3.1"
-description = "Pyap is an MIT Licensed text processing library, written in Python, for detecting and parsing addresses. Currently it supports USA, Canadian and British addresses."
-authors = ["Vladimir Goncharov <vladimarius@gmail.com>"]
-documentation = "https://github.com/vladimarius/pyap"
+description = "Pyap is an MIT Licensed text processing library, written in Python, for detecting and parsing addresses. Currently it supports USA, Canadian and British addresses. This is a fork maintained by Beauhurst."
+authors = ["Vladimir Goncharov <vladimarius@gmail.com>", "Beauhurst <dev@beauhurst.com>"]
+documentation = "https://github.com/Beauhurst/pyap"
 license = "MIT"
+packages=[{ include = "pyap_beauhurst" }]
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7.1",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3 :: Only",
+]
 readme="README.rst"
 
 [tool.poetry.dependencies]
-python = ">=2.7"
+python = "^3.7.1"
+pandas = "^1.2"
```

Comment:
does it need pandas?

Reply:
to parse some postcodes out of a csv of postcode data - so probably not. then we could have 3.11 perhaps too

Reply:
i didn't see any code for that here? are you trying to add new tests with these?

Reply:
https://github.com/Beauhurst/pyap/pull/1/files#diff-bd5b4b77e772420f99299f6025bc0818697bf855e8dce30b86ac7e57c6887849R441-R464

Reply:
:panda_face: :x: cb10c88
:snake: :three: :one: :one: :heavy_check_mark: 46a60c3